libs/platform/src/lib/components/form/checkbox-group/checkbox-group.component.ts

```diff
@@ -33,6 +33,18 @@ import { FormFieldControl } from '../form-control';
     providers: [{ provide: FormFieldControl, useExisting: CheckboxGroupComponent, multi: true }]
 })
 export class CheckboxGroupComponent extends CollectionBaseInput {
+    /**
+     * CBG label for accessibility
+     */
+    @Input()
+    ariaLabel: string;
```

Comment:
Let's move this up. do we have some usecase  aribaLabels also for component inheriting BaseComponent ?

Reply:
Moved these to base class created for all components. 
Components other than form components will also need these aria-* tags for accessibility.
Done